cmd/root.go

```diff
@@ -146,6 +146,9 @@ func loadDefaultManifestFileFromProjectPath(command string, projectPath string)
 	} else {
 		stderr = wski18n.T(wski18n.ID_ERR_MANIFEST_FILE_NOT_FOUND_X_path_X,
 			map[string]interface{}{wski18n.KEY_PATH: projectPath})
+		if cmd != nil {
+			cmd.Help()
```

Comment:
With these changes, the help message is displayed before the actual error which seems out of order.

![image](https://user-images.githubusercontent.com/206285/40691463-d3c95c62-6360-11e8-88d3-bcbca12386bf.png)

Can we change this to the following way of combining usage and the error:

```
if cmd != nil {
			stderr = stderr + wski18n.T(cmd.UsageString())
		}
```

This is how it appears:

![image](https://user-images.githubusercontent.com/206285/40691534-3286ba9c-6361-11e8-9ba6-6c6b327781cb.png)

But with this, everything shows in red as an error, please change it so that its displayed as a regular text.